dotnet/src/SemanticKernel/Planning/PlannerConfigBase.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+namespace Microsoft.SemanticKernel.Planning;
```

Comment:
Small formatting improvement:
```suggestion

namespace Microsoft.SemanticKernel.Planning;
```

Reply:
I thought `dotnet format` took care of this before... :/